tests/test_formatters.py

```diff
@@ -191,6 +191,17 @@ def test_escaping_govuk_in_email_templates(template_content, expected):
     assert expected in str(HTMLEmailTemplate({'content': template_content, 'subject': ''}))
 
 
+@pytest.mark.parametrize(
+    "subject,expected", [
+        ("bonjour | hi", "bonjour | hi"),
+        ("bonjour .", "bonjour."),
+        ('double -- dash', 'double \u2013 dash'),
+    ]
+)
+def test_subject_is_cleaned_up(subject, expected):
```

Comment:
I should have written this test in the previous PR. I was confident there were no bugs in the other methods used to clean up this field, I was wrong